### ISSUE TITLE
30 minute offset is added to selected time for different timezone on iOS devices

### ISSUE THREAD

**Issue by maheshp27** [skwasjer/skwas-cordova-plugin-datetimepicker#26]
Hi,

I am using this plugin for time selection purpose. It works perfectly fine on Android devices regardless of any timezone.
But on iOS devices, for some different timezone (in my case facing issue for Malaysia time zone) other  GMT+5.30: Indian standard time, picker adds 30 mins offset in selected time.
e.g If i selected 2:00 hrs on iOS devices, it gives 2:30 as selected time.

please help.

Thanks in advance.

**Comment by skwasjer** [comment]
I suspect it is related to (wrong) conversion of the date to a calendar instance (gregorian). It will need some research.

It might even be desirable to be able to specify timezone information as a plugin option.

**Comment by fragfeaster777** [comment]
Phonegap 8.2.2
Android 7.1.4

Hello,
I also have an offset issue but affecting my Android devices (don't have an iOS device at hand to test). On my Samsung S7 test device (Android 7.0), the offset is 33 minutes, ie if I set the plugin's time to 0700, the time returned is 0627. On an emulator running Android 9.0, the offset is 1 minute, ie if I set the plugin's time to 0700, the time returned is 0659. 

Happy to help debug in any way I can. 

Thanks and keep up the fantastic work!
